src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -970,8 +971,9 @@ private void startContainer() {
                 }
             }
 
+            info("Starting Docker image...");
             String startContainerCommand = getContainerCommand();
-            debug("startContainer, cmd="+startContainerCommand);
+            info(startContainerCommand);
```

Comment:
The getContainerCommand() already prints the docker run command here: https://github.com/OpenLiberty/ci.common/blob/master/src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java#L1190

Either remove the one from getContainerCommand() or remove this one to avoid printing it twice.  I'd prefer to remove the one from getContainerCommand() to be consistent with how we print the docker build command above.

Reply:
I believe that was done already.

Reply:
Thanks, I didn't notice that.